src/main/java/com/comphenix/protocol/wrappers/BukkitConverters.java

```diff
@@ -564,6 +596,10 @@ public static EquivalentConverter<WrappedProfileKeyData> getWrappedPublicKeyData
 		return ignoreNull(handle(WrappedProfileKeyData::getHandle, WrappedProfileKeyData::new, WrappedProfileKeyData.class));
 	}
 
+    public static EquivalentConverter<WrappedLoginSignature> getWrappedSignatureConverter() {
```

Comment:
indentation here and a few other places looks a little off. could just be the web editor being weird though

Reply:
Well in this case it's same as the other methods. However, I think I know what you mean. I adopted the checkstyle format, so you might noticed the following breaks if the line is too long.

```java
public void tooLong(String param1,
                    String param2) {
```

or 

```java
doSomething(
    param1,
    param2
);
```

Should I remove that?